src/IntegralArrays.jl

```diff
@@ -1,5 +1,78 @@
 module IntegralArrays
 
 # package code goes here
+using Images, ClosedIntervals
```

Comment:
That's a possibility, but I think it's a little dangerous because I suspect Jeff is mulling over whether we should ditch `CartesianIndex` and just use tuples for multidimensional indexing. (Certainly, I've given that some serious consideration.) If that happened, then indexing an IntegralImage with a 2-tuple would suddenly become very confusing.

So I think we need a type. Given the paralysis surrounding intervals, let's try making the nth package, I bet it will [suddenly become the standard](https://xkcd.com/927/).

The hardest part will be the package name. Some thoughts:
- SimpleIntervals
- LRIntervals
- OrderedIntervals
- Intervulz :smile:
- IntervalSets
- EnclosedIntervals

Of all these I think I like `IntervalSets` the best. It suggests that no arithmetic is supported. We might have the types `Closed`, `Open`, `LOpen`, and `ROpen` eventually (but perhaps start with `Closed`).

(Interestingly, there's a scala package with that name: https://github.com/rklaehn/intervalset)